front/src/styles/scss/common/components/_pathfinding.scss

```diff
@@ -91,6 +91,12 @@
     }
   }
 }
+.via,
+.place {
+  &.invalid-path-item {
+    color: #e05206;
```

Comment:
Do you think you can use a color from the theme ? You can use theme('colors.orange') for instance (if Thibaut defined an orange color), or directly var(--orange)

Reply:
done